build_runner/pubspec.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
   js: ^0.6.1+1
   logging: ^0.11.2
   meta: ^1.1.0
-  mime: ^0.9.3
+  mime: ^0.9.3+3
```

Comment:
why do this?

Reply:
guessing https://pub.dev/packages/mime#0963